nan_object_wrap.h

```diff
@@ -49,10 +58,30 @@ class ObjectWrap {
 
 
  protected:
+  // These fields must match internal v8 EmbedderDataSlot Indicies.
+  // unfortunately they are not centralized anywhere so we have to make our own
+  // https://source.chromium.org/chromium/chromium/src/+/main:v8/src/heap/embedder-tracing.h;l=176
+  // https://source.chromium.org/chromium/chromium/src/+/main:gin/public/wrapper_info.h;l=19
```

Comment:
I typically try and find permalinks for URLs that will be committed as comments, not sure if chromium makes that easy to grab or not.